influxql/parser.go

```diff
@@ -577,6 +577,15 @@ func (p *Parser) parseSelectStatement(tr targetRequirement) (*SelectStatement, e
 		return nil, err
 	}
 
+	// Set if the query is a raw data query or one with an aggregate
+	stmt.IsRawQuery = true
```

Comment:
In any event, would it more robust if `IsRawQuery` was a method, and not a flag? Yes, it means we might have to execute the method a couple of times, but it seems to me to be more coherent if `IsRawQuery` didn't require a SELECT statement to be brought into existence via parsing for `IsRawQuery` to be correct.